SDC/DataGenerator/datagenerator_deprecated.js

```diff
@@ -0,0 +1,130 @@
+const faker = require('faker');
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+var writer = csvWriter();
+var writer1 = csvWriter();
+var writer2 = csvWriter();
```

Comment:
Is this file deprecated because you wrote the other data generation file?

Reply:
Yes.